modules/Microsoft.DigitalTwins/digitalTwinsInstances/.test/common/dependencies.bicep

```diff
@@ -107,60 +103,55 @@ resource eventGridDomain 'Microsoft.EventGrid/domains@2022-06-15' = {
     properties: {
         disableLocalAuth: false
     }
-}
 
-resource eventGridTopic 'Microsoft.EventGrid/domains/topics@2022-06-15' = {
-    name: eventGridTopicName
-    parent: eventGridDomain
+    resource topic 'topics@2022-06-15' = {
+        name: 'topic'
+    }
 }
 
-resource evhrbacAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+resource eventHubNamespaceRbacAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
     name: guid(managedIdentity.id, 'evhrbacAssignment')
     scope: eventHubNamespace
     properties: {
-        roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '2b629674-e913-4c01-ae53-ef4638d8f975')
+        roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '2b629674-e913-4c01-ae53-ef4638d8f975') //Azure Event Hubs Data Sender
         principalId: managedIdentity.properties.principalId
         principalType: 'ServicePrincipal'
     }
 }
 
-resource sbhrbacAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+resource serviceBusRbacAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
     name: guid(managedIdentity.id, 'sbrbacAssignment')
     scope: serviceBus
     properties: {
-        roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '69a216fc-b8fb-44d8-bc22-1f3c2cd27a39')
+        roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '69a216fc-b8fb-44d8-bc22-1f3c2cd27a39') //Azure Service Bus Data Sender
         principalId: managedIdentity.properties.principalId
         principalType: 'ServicePrincipal'
     }
 }
 
 @description('The resource ID of the created Virtual Network Subnet.')
-output subnetResourceId string = virtualNetwork.properties.subnets[0].id
+output subnetResourceResourceId string = virtualNetwork.properties.subnets[0].id
 
 @description('The principal ID of the created Managed Identity.')
-output managedIdentityPrincipalId string = managedIdentity.properties.principalId
+output managedIdentityPrincipalResourceId string = managedIdentity.properties.principalId
 
 @description('The resource ID of the created Private DNS Zone.')
-output privateDNSResourceId string = privateDNSZone.id
+output privateDNSResourceResourceId string = privateDNSZone.id
 
 output eventhubNamespaceName string = eventHubNamespace.name
 
-output eventHubId string = eventHub.id
+output eventHubResourceResourceId string = eventHubNamespace::eventHub.id
```

Comment:
It appears that you accidently duplicated `Resource` in some instances like above:
`eventHubResourceResourceId` should be `eventHubResourceId`.

Please replace all these occurences. Should be as easy as a Search & Replace of all `ResourceResourceId` with `ResourceId`.